mir-module/src/main/resources/xsl/response-mir-utils.xsl

```diff
@@ -132,7 +132,7 @@
     <xsl:variable name="label.previousPage" select="i18n:translate(concat($i18nprefix, '.previous'), $page - 1)" />
     <xsl:variable name="label.nextPage" select="i18n:translate(concat($i18nprefix, '.next'), $page + 1)" />
 
-    <ul id="{$id}-paginate" class="pagination {$class} d-inline-flex">
```

Comment:
Seems the combination of text-center of the parent element and display inline-block of the list element is not longer possible, because pagination was set to display flex. To prevent the full width of the list and to enable centred alignment it has to be a inline element again. Someone used the d-inline-flex class for that. If you remove this, the pagination is not longer centred.